katcp/kattypes.py

```diff
@@ -151,17 +151,14 @@ class Int(KatcpType):
         The minimum allowed value. Ignored if not given.
     max : int
         The maximum allowed value. Ignored if not given.
-
     """
-
```

Comment:
This blank line should remain - it's prescribed by PEP 257.

Reply:
Addressed.